paasta_tools/marathon_tools.py

```diff
@@ -999,15 +999,16 @@ def is_old_task_missing_healthchecks(task, marathon_client):
     return False
 
 
-def get_num_at_risk_tasks(app):
+def get_num_at_risk_tasks(app, draining_hosts=None):
```

Comment:
Can you make this a required parameter?

I don't like it when functions kinda do things on their own when it "should" be the responsibility of the caller to inject the dependency.

Additionally it will stop us from making this performance mistake again!